app/assets/stylesheets/styles.scss

```diff
@@ -2,6 +2,8 @@ $junk_color: #6e6e6e;
 $heirloom_color: #04b404;
 $antiquity_color: #08298a;
 $relic_color: #ff8000;
+$find_panel_bg: #d8d8d8;
```

Comment:
Name of variable `find_panel_bg` should be written in all lowercase letters with hyphens instead of underscores